plugin-server/src/main/ingestion-queues/session-recording/services/session-manager.ts

```diff
@@ -171,14 +171,23 @@ export class SessionManager {
                 message.metadata.timestamp
             )
 
-            const messageData = convertToPersistedMessage(message)
-            this.setEventsRangeFrom(message)
+            const content =
+                convertForPersistence(message.eventsByWindowId)
+                    .map((x) => JSON.stringify(x))
```

Comment:
i bet this is where we should be escaping newlines in console logs or something 🤔

Reply:
just thinking out loud though not suggesting we do it now